apps/condo/domains/ticket/schema/TicketFile.js

```diff
@@ -37,6 +39,25 @@ const TicketFile = new GQLListSchema('TicketFile', {
         },
     },
     hooks: {
+        afterChange: async ({ updatedItem, listKey }) => {
+            if (updatedItem) {
+                const { id, file } = updatedItem
+                if (file) {
+                    const { filename } = file
+                    const key = `${TICKET_FILE_FOLDER_NAME}/${filename}`
+                    // OBS will lowercase all keys from meta
+                    const metaToSet = {
+                        listkey: listKey,                    
+                        id,
+                    }
+                    try {
+                        await Adapter.acl.setMeta(key, metaToSet)
+                    } catch (error) {
+                        console.error('OBS set meta for file', error)
```

Comment:
It's better not to hide the error. We should not save the file in such cases

Reply:
Done!